lms/templates/courseware/gradebook.html

```diff
@@ -74,13 +74,24 @@
         <thead>
           <tr> <!-- Header Row -->
             %for section in templateSummary['section_breakdown']:
-              <th><div class="assignment-label">${section['label']}</div></th>
+              <%
+                tooltip_str = section['detail']
+                # We are making header labels from the first student record. So for tool tip (title),
+                # I am processing this string ```section['detail']``` from student record and removing
+                # all student related data i.e marks, percentage etc to get only the title of homework.
+                if "=" in section['detail']:
+                    tooltip_str = section['detail'][0: section['detail'].rfind('=')]
+
+                if "-" in tooltip_str:
+                    tooltip_str = tooltip_str[0: tooltip_str.rfind('-')]
+              %>
+              <th title="${tooltip_str}"><div class="assignment-label">${section['label']}</div></th>
             %endfor
-            <th><div class="assignment-label">Total</div></th>
+            <th title="${_('Total')}"><div class="assignment-label">Total</div></th>
```

Comment:
Found one other thing, actually. This line has the word "Total" twice, but only one of those instances is internationalized. They both should be internationalized.